### PR TITLE
[WIP] Adding the docs contribution guide

### PR DESCRIPTION
**Pull request by lenadoc**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+```bash
+arch -arch x86_64 bundle install
+```
+
+{% endinfo_block %}
+
+## Run the documentation site locally
+
+Now that you have Jekyll and the documentation site installed, you can run the site locally. Do the following:
```
Comment (andriitserkovnyi) [inline docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:121]:
```suggestion
To run Spryker Documentation locally, do the following:
```

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+1. Go to the local documentation site directory:
+```bash
+cd spryker-docs
+```
+2. Run:
```
Comment (andriitserkovnyi) [inline docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:127]:
```suggestion
2. Build the application:
```

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+```bash
+bundle exec jekyll serve
+```
+Now, you can access the local copy of the Spryker documentation site at http://localhost:4000.
+
+{% info_block infoBox "MacOS on M1 processor" %}
+
+If you use the latest generation of MacBooks running on M1 processors,  prefix this command like this:
```
Comment (andriitserkovnyi) [inline docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:135]: Adjust it similarly to the previous notel

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+{% endinfo_block %}
+ 
+## Tips and tricks
+
+* To regenerate only those pages that were added or updated since the last build, use this command:
+```bash
+bundle exec jekyll serve --incremental
+```
+* To automatically refresh the page with each change you make to the source files, use this command:
```
Comment (andriitserkovnyi) [inline docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:149]: Does this command work If I'm already running the app?

**Review by andriitserkovnyi — CHANGES_REQUESTED** [review]

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
+---
+title: Building the documentation site
```
[the remark this answers is not in the document]
Comment (lenadoc) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:2]: Building is a broader term, and running is the sub-process of building, also included as the separate item in this documentation. Jekyll, readthedocs, magent, [github Pages](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll), etc. also use the "building" term, so I would rather leave it.

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+# If you're using Bash
+echo 'export PATH="$HOME/.gem/ruby/X.X.0/bin:$PATH"' >> ~/.bash_profile
+
+# Not which Shell you are using? Type
+echo $SHELL
+```
+### Windows
+
+To install Jekyll on Windows, follow the [official Jekyll on Windows documentation](https://jekyllrb.com/docs/installation/windows/).
```
[the remark this answers is not in the document]
Comment (lenadoc) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:86]: This is how the article is called on the Jekyll website, might be better to stick to the original article name

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+### Ubuntu
+
+To install Jekyll on Ubuntu, follow the [official Jekyll on Ubuntu documentation](https://jekyllrb.com/docs/installation/ubuntu/).
```
[the remark this answers is not in the document]
Comment (lenadoc) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:92]: same as for windows

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md (hunk 4 above)
Comment (lenadoc) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:151]: no, it's an independent command. You should first stop the previous process to run the new one

Comment (andriitserkovnyi) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:150]: Then make it clear by replacing `use this command` with `build the site by running the following command`.

**Review by andriitserkovnyi — DISMISSED** [review]

File: docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/scos/user/intro-to-spryker/contributing-to-documentation/building-the-documentation-site.md:2]: Good one, you are right :)

**Pull request merged**